nixos/modules/system/boot/luksroot.nix

```diff
@@ -894,6 +896,19 @@ in
             '';
           };
         };
+
+        config = mkIf (clevis.enable && (hasAttr name clevis.devices)) {
+          preOpenCommands = mkIf (!systemd.enable) ''
+            mkdir -p /clevis-${name}
```

Comment:
No need to `rmdir` this, right? It goes away when root is switched?

Reply:
Yes